specification/communication/data-plane/Identity/preview/2025-04-01-preview/examples/CreateIdentity.json

```diff
@@ -0,0 +1,44 @@
+{
+  "parameters": {
+    "api-version": "2025-04-01-preview",
+    "content-type": "application/json",
+    "endpoint": "https://my-resource.communication.azure.com",
+    "body": {
+      "externalId": "sally@contoso.com",
+      "createTokenWithScopes": [
+        "chat",
+        "voip",
+        "chat.join",
+        "chat.join.limited",
+        "voip.join"
+      ],
+      "expiresInMinutes": 60
+    }
+  },
+  "responses": {
+    "201": {
+      "body": {
+        "identity": {
```

Comment:
I'm wondering if we should return `lastTokenIssuedAt` here as well, then we'd have `identity` be the same schema as on the GET route. But it would tie them together and GET identity will maybe later return other Entra properties that aren't relevant to BYOI. Could still be okay as they will be optional.